src/schema/v2/__tests__/filterArtworksConnection.test.js

```diff
@@ -268,6 +268,64 @@ describe("filterArtworksConnection", () => {
     })
   })
 
+  describe(`Returns proper pagination information`, () => {
```

Comment:
Just a heads up, this won't be run yet in CI. I should be done w/ the fix tests ticket today, but until that's finished it's ignored.

Reply:
Yep! That's how the passing `...on Node` test below in this file (which was passing locally!), started to break after a later commit changed a default export to a named one. This was just for my own sanity to run locally.